<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot name="hatch_door">
  <link name="door_link">
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <box size="0.4 0.3 0.02"/>
      </geometry>
    </collision>

    <collision>
        <origin xyz="0.17 0 -0.019" rpy="0 0 0" />
        <geometry>
            <box size="0.018 0.29 0.02"/>
        </geometry>
    </collision>

    <collision>
        <origin xyz="-0.17 0 -0.019" rpy="0 0 0" />
        <geometry>
            <box size="0.018 0.29 0.02"/>
        </geometry>
    </collision>

    <inertial>
      <origin xyz="0 0 0.0" rpy="0 0 0" />
      <mass value="5" />
      <inertia ixx="1" ixy="0.0" ixz="0.0"
                       iyy="1"   iyz="0.0"
                                 izz="1" />
    </inertial>
  </link>

  <link name="handle_link">
    <inertial>
      <origin xyz="0 0 0.0" rpy="0 0 0" />
      <mass value="0.2" />
      <inertia ixx="0.00067" ixy="0.0"     ixz="0.0"
                             iyy="0.00071" iyz="0.0"
                                           izz="0.000048" />
    </inertial>

    <collision>
      <origin xyz="-0.05 0 0" rpy="0 0 0" />
      <geometry>
        <box size="0.02 0.02 0.16"/>
      </geometry>
    </collision>

    <collision>
      <origin xyz="-0.03 0 0.09" rpy="0 0 0" />
      <geometry>
        <box size="0.06 0.02 0.02"/>
      </geometry>
    </collision>

    <collision>
      <origin xyz="-0.03 0 -0.09" rpy="0 0 0" />
      <geometry>
        <box size="0.06 0.02 0.02"/>
      </geometry>
    </collision>
  </link>

  <joint name="handle_joint" type="fixed">
    <origin xyz="0 -0.12 0.01" rpy="0 1.57 0" />
    <parent link="door_link" />
    <child link="handle_link" />
  </joint>
</robot>
